sys_conn_df_darwin.go

```diff
@@ -0,0 +1,67 @@
+//go:build darwin
+
+package quic
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/quic-go/quic-go/internal/utils"
+)
+
+func setDF(rawConn syscall.RawConn) (bool, error) {
+	uname := &unix.Utsname{}
+	err := unix.Uname(uname)
+	if err != nil {
+		return false, err
+	}
+	if !isAtLeastOS11(uname) {
```

Comment:
Can you add a comment here why we're testing for this?

Reply:
I picked this up from chromium source code and also some random answer on stackoverflow / github. I couldn't find a more official source so I've just added a comment saying it's only supported from macOS 11.

Reply:
Could you link the Chromium source code and / or the Stackoverflow here?

Reply:
done